tools/go.sum

```diff
@@ -134,11 +135,15 @@ github.com/go-logfmt/logfmt v0.4.0/go.mod h1:3RMwSq7FuexP4Kalkev3ejPJsZTpXXBr9+V
 github.com/go-logr/logr v0.1.0/go.mod h1:ixOQHD9gLJUVQQ2ZOR7zLEifBX6tGkNJF4QyIY7sIas=
 github.com/go-logr/logr v0.2.0/go.mod h1:z6/tIYblkpsD+a4lm/fGIIU9mZ+XfAiaFtq7xTgseGU=
 github.com/go-ole/go-ole v1.2.1/go.mod h1:7FAglXiTm7HKlQRDeOQ6ZNUHidzCWXuZWq/1dTyBNF8=
+github.com/go-openapi/jsonpointer v0.0.0-20160704185906-46af16f9f7b1/go.mod h1:+35s3my2LFTysnkMfxsJBAMHj/DoqoB9knIWoYG/Vk0=
```

Comment:
Do you understand why it was added?
I don't see it as part of:
https://github.com/buildpacks/imgutil/commit/cc867729ddde7cefe3c08e3b714e34856a7cc67b.

Reply:
My answer for this is: go tooling is confusing.

It comes from ggcr. I think the latest ggcr update brought it in, but was `go mod tidy` wasn't run in the `tools/*` module during that PR.